src/main/java/core/basesyntax/data/input.csv

```diff
@@ -1,4 +1,4 @@
-type,fruits,quantity
+src/main/resources
```

Comment:
On github I can't comment on files, so I left a comment on the first line, that's why you probably got confused. Undo this change, first line was correct - it sets names to the columns in CSV format. Just move this file physically from src/main/java/core/basesyntax/data/ to src/main/resources